pkg/executors/executor.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 type Executor interface {
-	Prepare() int
+	Prepare(bool) int
```

Comment:
The kvm bool flag makes sense for DockerComposeExecutor, but for the ShellExecutor is not valid.

Instead of modifying the interface, I would vote for modifying the constructor of the concretion, aka the NewDockerComposeExecutor function.

Reply:
If that is not possible, let's find some other way of injecting this in a way that it makes sense for both shell and docker executors.

Reply:
yeah, that makes more sense, I'll do that 🙇‍♂️

Reply:
Done